variables.tf

```diff
@@ -195,100 +195,100 @@ EOF
 }
 
 variable "hms_nofile_ulimit" {
-  description = "ulimit for the metastore container"
+  description = "Ulimit for the Hive Metastore container."
   type        = "string"
   default     = "32768"
 }
 
 variable "hms_docker_image" {
-  description = "docker image id for the hive metastore"
+  description = "Docker image ID for the Hive Metastore."
   type        = "string"
 }
 
 variable "hms_docker_version" {
-  description = "version of the docker image for the hive metastore"
+  description = "Version of the Docker image for the Hive Metastore."
   type        = "string"
 }
 
 variable "hms_readwrite_instance_count" {
-  description = "desired count of the RW hive metastore service"
+  description = "Desired instance count of the read/write Hive Metastore service."
   type        = "string"
   default     = "2"
 }
 
 variable "hms_readonly_instance_count" {
-  description = "desired count of the RO hive metastore service"
+  description = "Desired instance count of the read only Hive Metastore service."
   type        = "string"
   default     = "2"
 }
 
 variable "apiary_s3_alarm_threshold" {
-  description = "will trigger cloudwatch alarm if s3 is greater than this, default 1TB"
+  description = "Threshold number of bytes to trigger Cloudwatch alarm if size of data in S3 is greater than this. Default is 1TB."
   type        = "string"
-  default     = "10000000000000"
+  default     = "1000000000000"
```

Comment:
@rpoluri @pradeepbhadani The default value here previously was actually 10TB in bytes, but the documentation said 1TB. I'm not sure which is correct. I've made the change to the number of bytes to be 1TB but perhaps the change needs to be the other way around? I'm also not clear from reading this what S3 bucket size this is actually watching to trigger an alert, can someone please clarify.

Reply:
As far as I can tell this is unused.

```
sh-3.2$ grep -r -i apiary_s3_alarm_threshold *
variables.tf:variable "apiary_s3_alarm_threshold" {
sh-3.2$
```

Reply:
yes It is currently unused.
This was originally added to configure storage utilization alerts on S3 buckets, removed corresponding cloudwatch alert when added support for external buckets.

Reply:
We should remove it then.